APMrover2/AP_MotorsUGV.cpp

```diff
@@ -322,8 +324,8 @@ void AP_MotorsUGV::output_throttle(SRV_Channel::Aux_servo_function_t function, f
                 // do nothing
                 break;
         }
-        // brushed-with-relay motors should receive positive values
-        throttle = fabsf(throttle);
+        // invert the ouput to always have positive value calculated by calc_pwm
+        throttle = reverse_multiplier * fabsf(throttle);
```

Comment:
yes, it is needed because in calc_pwm(), the pwm value will be reverse according the Servo_reverse : https://github.com/ArduPilot/ardupilot/blob/master/libraries/SRV_Channel/SRV_Channel.cpp#L99 .
This ensure that the value is always positive !